ProcessMaker/processmaker
# FOUR-11843: Import/Export Screen Templates

As a user, I would like to easily Import and Export Screen Templates. This feature should allow for seamless integration and customization of user interfaces, enhancing the overall design and user experience.


## Import
- Navigate to Screen Templates Tab
- Find the Import option
- The Import Screen will display
- The following flow works exactly as Process Import.

## Export
- The flow works exactly as Screen Export.

## Solution
- Create Import/Export functionality of Screen Templates
- Implement unitTest

## How to Test
`phpunit tests/Feature/Templates/Api/ScreenTemplateTest.php`

1. Navigate to Screens 
2. Create a Screen Templates (My Templates and Public)
3. Navigate to My Templates or Public Templates
4. In the Ellipsis menu, you should see an option to Export Template
5. Follow the instructions to Export
6. You should be able to get the downloaded manifest.

## Now you can import the Screen Template

1. Navigate to Screens 
2. Create a Screen Templates (My Templates and Public)
3. Click on Import 
4. Select the Screen Template file
5. Click on Import
6. You should be able to see the new Screen Template in the System

## Related Tickets & Packages
- Ticket: [FOUR-11843](https://processmaker.atlassian.net/browse/FOUR-11843)
- 
ci:next

## Code Review Checklist


[FOUR-11843]: https://processmaker.atlassian.net/browse/FOUR-11843?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
